iox_client_management_test.go

```diff
@@ -7,14 +7,14 @@ import (
 )
 
 func ExampleClient_CreateDatabase() {
-	config, _ := influxdbiox.ClientConfigFromJSONString("localhost:8082")
+	config, _ := influxdbiox.ClientConfigFromAddressString("localhost:8082")
```

Comment:
`ClientConfigFromJSONString` expects, as its name implies, JSON input but this example has the address string